libs/sdk-ui-charts/src/highcharts/chart/HighChartsRenderer.tsx

```diff
@@ -199,6 +201,17 @@ export default class HighChartsRenderer extends React.PureComponent<
         });
     }
 
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    public onChartSelection(event: any) {
+        const chartWrapper = event.target.renderTo.parentElement;
+        const resetZoomButton = chartWrapper.closest(".gd-base-visualization").querySelector(".s-zoom-out");
```

Comment:
classnames with `s-` should be used for tests only

Reply:
Updated!